src/views/Settings.vue

```diff
@@ -6,38 +6,83 @@
       </ion-toolbar>
     </ion-header>
     <ion-content>
-      <ion-item>
-        <ion-icon :icon="businessOutline" slot="start" />
-        <ion-label>{{$t("eCom Store")}}</ion-label>
-        <ion-select interface="popover" :placeholder="$t('store name')" :value="currentFacility.facilityId" @ionChange="setFacility($event)">
-          <ion-select-option v-for="facility in (userProfile ? userProfile.facilities : [])" :key="facility.facilityId" :value="facility.facilityId" >{{ facility.name }}</ion-select-option>
-        </ion-select>
-      </ion-item>
-      <ion-item>
-        <ion-icon :icon="codeWorkingOutline" slot="start"/>
-        <ion-label>{{ $t("OMS") }}</ion-label>
-        <p slot="end">{{ baseURL ? baseURL : instanceUrl }}</p>
-      </ion-item>
-      <ion-item>
-        <ion-icon :icon="personCircleOutline" slot="start" />
-        <ion-label>{{ userProfile !== null ? userProfile.partyName : '' }}</ion-label>
-        <ion-button slot="end" fill="outline" color="dark" @click="logout()">{{ $t("Logout") }}</ion-button>
-      </ion-item>
+      <div class="user-profile">
+        <ion-card>
+          <ion-item lines="full">
+            <ion-avatar slot="start" v-if="userProfile?.partyImageUrl">
+              <Image :src="userProfile.partyImageUrl"/>
+            </ion-avatar>
+            <ion-label>
+              {{ userProfile?.partyName }}
+              <p>{{ userProfile?.userLoginId }}</p>
+            </ion-label>
+          </ion-item>
+          <ion-button fill="outline" color="danger" @click="logout()">{{ $t("Logout") }}</ion-button>
+          <!-- Commenting this code as we currently do not have reset password functionality -->
+          <!-- <ion-button fill="outline" color="medium">{{ $t("Reset password") }}</ion-button> -->
+        </ion-card>
+      </div>
+      <h1>{{ $t('OMS') }}</h1>
+      <section>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-subtitle>
+              {{ $t("OMS instance") }}
+            </ion-card-subtitle>
+            <ion-card-title>
+              {{ baseURL ? baseURL : instanceUrl }}
+            </ion-card-title>
+          </ion-card-header>
+          <ion-card-content>
+            {{ $t('This is the name of the OMS you are connected to right now. Make sure that you are connected to the right instance before proceeding.') }}
+          </ion-card-content>
+          <ion-button @click="goToOms" fill="clear">
+            {{ $t('Go to OMS') }}
+            <ion-icon slot="end" :icon="openOutline" />
+          </ion-button>
+        </ion-card>
+        <ion-card>
+          <ion-card-header>
+            <ion-card-subtitle>
+              {{ $t("Shopify Config") }}
```

Comment:
We do not have ecommerce or shopify config in this app, we have facilities that the user is associated with. Title and content should be accordingly

Reply:
Sure sir. Updating it accordingly.